src/utils/send-reset-email.js

```diff
@@ -0,0 +1,42 @@
+import sgMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
+import mailgen from 'mailgen';
+
+dotenv.config();
+
+sgMail.setApiKey(process.env.sendgridKey);
+const hostUrl = `http://${process.env.BASE_URL}`;
+const mailGenerator = new mailgen({
+  theme: 'default',
+  product: {
+    name: 'barefoot nomad',
+    link: 'barefoot nomad'
+  }
+});
+const sendMsg = (newUserEmail, token, firstName, content) => {
+  const email = mailGenerator.generate({
```

Comment:
Let's have a single way to have our emails look like @Ntare22 @izzett222

Reply:
we changed the send email function and the email are similar

Reply:
Let's also refactor the welcome email to this format

Reply:
we already changed the welcome email that Jim created by using the new send email function